### ISSUE TITLE
xASL_quant_SinglePLD: Check "merge conflict" #391

### ISSUE THREAD

**Issue by HenkMutsaerts** [ExploreASL/ExploreASL#521]
Jan would you mind checking this code:

https://github.com/ExploreASL/ExploreASL/blob/feature-%23391_BASIL_Implementation/Functions/xASL_quant_SinglePLD.m

at line 103 and 286. These are essentially merging conflicts ;) which is why I wouldn't make this a new issue, not to complicate history!

I would like to have this "clean" before starting the PR.

**Comment by jan-petr** [comment]
I agree that this new issue is not really a good solution, though not sure how you propose to solve it :)

We have the code in develop like it is, and we need it to stay like it is - for Multi-band and other reasons (BIDS etc). If BASIL doesn't like it, then we have to make a specific solution for BASIL.

So I agree to close this issue and then we  either need to 
1) rebase the BASIL branch onto the new develop. Fix conflicts, and then write this BASIL adjustment. Before merging the BASIL branch.

2) Or (my preferred solution) merge the BASIL branch while fixing conflicts and make a new issue to be immediately do the proposed adjustments.

How do we proceed?
But we can't fix the problem here and we can't revert the changes made.

**Comment by HenkMutsaerts** [comment]
Can you please have a look at the actual function that I mention? Then you will see that there are two sections that I marked for you to check. 

What we have now works, which is simply stating `if x.bUseBasilQuantification` then use the old code, otherwise your new code. But you might very easily see that the old code for Basil also can be replaced by your new code. Basil simply uses imSliceNumber as `x.Q.SliceReadoutTime` without applying this to `ScaleImage`

**Comment by jan-petr** [comment]
Fixed directly in #391.
